eclass/gnome2-utils.eclass

```diff
@@ -123,8 +119,8 @@ gnome2_gconf_savelist() {
 # using gconftool-2.
 # This function should be called from pkg_postinst.
 gnome2_gconf_install() {
-	has ${EAPI:-0} 0 1 2 && ! use prefix && EROOT="${ROOT}"
-	local updater="${EROOT}${GCONFTOOL_BIN}"
+	[[ ${EAPI:-0} == [0123456] ]] && BROOT=${EROOT}
```

Comment:
Loses proper support for EAPI-2 and below, as advertised in commit log. But I'm not convinced that's fine if old EAPIs are still supported officially as per EAPI guard at top in line 22. Again maybe not a concern if going to $PATH reliance.

Reply:
Yep. But according to https://qa-reports.gentoo.org/output/eapi-per-eclass/gnome2-utils.eclass/ the lowest EAPI used is 4 so the check and adjustment is also unneeded in-tree.